### ISSUE TITLE
Can't mine stone

### ISSUE THREAD

**Issue by DayIsGone** [qejmc/Better-Bricks#1]
With the 1.19 beta installed I can no longer mine stone with a pickaxe. Wood and Stone takes awhile to break the blocks gold, netherrite, iron, diamond just instadestroy the block.

I even removed all my mods to see if there was an issue between them all causing it

**Comment by qejmc** [comment]
From my testing the brick blocks all have the correct breaking speeds. If by 'stone' you are referring to Minecraft stone, this too breaks at the correct rate (not sure why the mod would affect that). If this issue persists, could you please give step-by-step instructions on how to replicate this bug?

**Comment by DayIsGone** [comment]
I loaded your mod on a game with 1.19-forge-41.0.17 installed. I spawned on a stony cliff so I made wooden pickaxe to mine some stone so I could collect the iron near by and the brick didn't drop anything when the mining animation was finished.

at first I did have other mods installed but I split them up until I narrowed it down to your mod causing the glitch as I  could mine with the others installed.

I did just go in game and record a video to show what I mean. https://youtu.be/JZhCtxdmusc
